cloud/blockstore/libs/storage/partition_nonrepl/part_nonrepl_rdma_actor.cpp

```diff
@@ -121,6 +122,27 @@ bool TNonreplicatedPartitionRdmaActor::InitRequests(
         return false;
     }
 
+    if (!msg.Record.GetHeaders().GetIsBackgroundRequest()
+            && RequiresReadWriteAccess<TMethod>
+            && PartConfig->IsReadOnly())
+    {
+        reply(
+            ctx,
+            requestInfo,
+            PartConfig->MakeIOError(
+                "disk in error state",
+                true // cooldown passed
```

Comment:
Looks like in regular actor this cooldown also makes errors from a broken disks silent for a while. We may want to replicate that behavior in rdma actor. Not necessarily in this PR

Reply:
Yep I was thinking of doing the broken disk logic in separate pr